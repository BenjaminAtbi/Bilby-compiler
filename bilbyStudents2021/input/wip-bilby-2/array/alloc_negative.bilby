main {
	imm b := alloc[int](-4);
}