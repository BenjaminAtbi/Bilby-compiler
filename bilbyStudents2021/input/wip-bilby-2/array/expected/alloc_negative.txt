Runtime error: array has negative length
